var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -46,6 +46,7 @@ class Binutils(Package):
     # OpenSpeedShop and cbtf-krell
     variant('krellpatch', default=False,
             description="build with openspeedshop based patch.")
+    variant('plugins', default=False)
```

Comment:
> Sorry I don't have one

you don't have to have it, just add one :smile:
i.e. `enable plugins needed for gold linker`.